src/ApiException.php

```diff
@@ -162,14 +162,23 @@ public static function createFromRpcStatus(Status $status)
      * Creates an ApiException from a GuzzleHttp RequestException.
      *
      * @param RequestException $ex
+     * @param boolean $stream
      * @return ApiException
      * @throws ValidationException
      */
-    public static function createFromRequestException(RequestException $ex)
+    public static function createFromRequestException(RequestException $ex, $stream = false)
     {
         $res = $ex->getResponse();
         $body = (string) $res->getBody();
-        if ($error = json_decode($body, true)['error']) {
+        $decoded = json_decode($body, true);
+        
+        // A streaming response body will return one error in an array. Parse
+        // that first (and only) error message, if provided.
+        if ($stream && isset($decoded[0])) {
```

Comment:
Super-nit, but can we change the variable name to `$isStream`? This will make it a little more obvious that it's a boolean

Reply:
Done!